ui-tests/src/test/java/org/openmrs/reference/page/HeaderPage.java

```diff
@@ -24,6 +25,7 @@ public void clickOnHomeLink() {
     }
 
     public void logOut() throws InterruptedException {
+    	waiter.until(ExpectedConditions.visibilityOfElementLocated(LOGOUT));
```

Comment:
The same here

Reply:
No identation to be applied here

Reply:
See my comment [here](https://github.com/openmrs/openmrs-distro-referenceapplication/pull/503/commits/20aa60c34874632e6109b53f2d9370bc6f5538d7#r675345648)